packages/blade/src/components/Input/BaseInput/StyledBaseInput.native.tsx

```diff
@@ -4,30 +4,88 @@ import styled from 'styled-components/native';
 import type { StyledBaseInputProps } from './StyledBaseInput.d';
 import { getBaseInputStyles } from './baseInputStyles';
 
-const StyledNativeBaseInput = styled.TextInput<StyledBaseInputProps & { isFocused: boolean }>(
-  (props) => ({
-    ...getBaseInputStyles({
-      theme: props.theme,
-      isFocused: props.isFocused,
-      isDisabled: !props.editable,
-      validationState: props.validationState,
-      leadingIcon: props.leadingIcon,
-      prefix: props.prefix,
-      interactionElement: props.interactionElement,
-      suffix: props.suffix,
-      trailingIcon: props.trailingIcon,
-    }),
+type StyledComponentAutoCompleteAndroid =
+  | 'off'
+  | 'name'
+  | 'cc-csc'
+  | 'cc-exp'
+  | 'cc-exp-month'
+  | 'cc-exp-year'
+  | 'cc-number'
+  | 'email'
+  | 'username'
+  | 'password'
+  | 'postal-code'
+  | 'tel'
+  | undefined;
+
+const autoCompleteSuggestionTypeAndroid = {
+  none: 'off',
+  name: 'name',
+  email: 'email',
+  username: 'username',
+  password: 'password',
+  newPassword: 'password-new',
+  oneTimeCode: 'sms-otp',
+  telephone: 'tel',
+  postalCode: 'postal-code',
+  countryName: 'postal-address-country',
+  creditCardNumber: 'cc-number',
+  creditCardCSC: 'cc-csc',
+  creditCardExpiry: 'cc-exp',
+  creditCardExpiryMonth: 'cc-exp-month',
+  creditCardExpiryYear: 'cc-exp-year',
+} as const;
+
+const autoCompleteSuggestionTypeiOS = {
```

Comment:
nit: 
```suggestion
const autoCompleteSuggestionTypeIOS = {
```

Reply:
fixed in b8929d24b4e7a0d218bba7e599803658e69dc4a8